src/addons/MarkerClusterer.js

```diff
@@ -89,9 +89,14 @@ export default class MarkerClusterer extends Component {
 
   render() {
     if (this.state.markerClusterer) {
+      let { children, ...restProps } = this.props;
+
       return (
-        <MarkerClustererCreator markerClusterer={ this.state.markerClusterer } { ...this.props }>
-          { this.props.children }
+        <MarkerClustererCreator
+          markerClusterer={this.state.markerClusterer}
+          {...restProps}
+        >
```

Comment:
I would deactivate in case active the rule `react/jsx-closing-bracket-location`, so closing brackets can be located everywhere.